src/main/java/de/mediathekview/mserver/crawler/srf/SrfConstants.java

```diff
@@ -0,0 +1,21 @@
+package de.mediathekview.mserver.crawler.srf;
+
+public interface SrfConstants {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Move constants to a class or enum. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1214)